modules/custom/az_dam/config/quickstart/core.entity_form_display.media.ib_dam_embed.default.yml

```diff
@@ -0,0 +1,62 @@
+uuid: 3e3c5ab9-8755-422a-a3db-661b855c0d2a
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.media.ib_dam_embed.field_media_ib_dam_embed
+    - media.type.ib_dam_embed
+  module:
+    - link
+    - path
+_core:
+  default_config_hash: 5z7JVAF7hj_6rtGjz8u6N_YkYqgESCg0-JIL93lY1ok
```

Comment:
We also omit these default config hash values from exported configs.
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.